build.sh

```diff
@@ -191,7 +191,7 @@ for ARG in "$@"; do
         --clang             ) useClang=1 ;;
         --intel             ) useIcc=1 ;;
         --cppcheck          ) cppcheck=1 ;;
-        --dev|--devel       ) flags="$flags -DCMAKE_BUILD_TYPE=Devel"   ; build="$root/build_dev";;
+        --dev|--devel       ) flags="$flags -DCMAKE_BUILD_TYPE=RelWithDebInfo"; build="$root/build_dev";;
```

Comment:
I'd prefer we keep the devel option as RelwithDebInfo feels like it should be release with extra sauce

Reply:
Done